src/visualization/Turi Create Visualization/src/user_interface/src/elements/Annotate/index.js

```diff
@@ -184,6 +168,26 @@ class Annotate extends Component {
     });
   }
 
+  setAnnotation = (rowIndex, labels) => {
+    const root = Root.fromJSON(messageFormat);
+    const ParcelMessage = root.lookupType("TuriCreate.Annotation.Specification.ClientRequest");
+    const payload = {"annotations": {"annotation":[{"labels": [{"stringLabel": labels}], "rowIndex": [rowIndex]}]}};
+    const err = ParcelMessage.verify(payload);
+    
+    if (err)
+      throw Error(err);
+
+    const message = ParcelMessage.create(payload);
+    const buffer = ParcelMessage.encode(message).finish();
+    const encoded = btoa(String.fromCharCode.apply(null, buffer));
+
+    if (window.navigator.platform == 'MacIntel') {
+      window.webkit.messageHandlers["scriptHandler"].postMessage({status: 'writeProtoBuf', message: encoded});
+    } else {
+      window.postMessageToNativeClient(encoded);
```

Comment:
Does not need to be done in this PR, but in the future, let's abstract away the Linux vs. macOS way we communicate back to the host.